CHECLabPy/spectrum_fitters/gentile.py

```diff
@@ -6,7 +6,7 @@
 
 
 class GentileFitter(SpectrumFitter):
-    def __init__(self, n_illuminations, config_path=None):
+    def __init__(self, n_illuminations, config_path=None, fit_range=None):
```

Comment:
The range for the fit can be edited by passing a config yaml file containing:

```
range:
- -0.5
- 4
```

Is this okay to use instead of these changes?

Reply:
Yeah that is no problem! I did not read through the code thoroughly enough to see that you can set the range through the config file.